modules/new_profile/test/new_profileTest.php

```diff
@@ -171,13 +172,22 @@ function testNewProfilePSCIDError() {
      * @return none
      */
     function testNewProfileDoBDateError() {
+$this->markTestSkipped("Config not properly set up to test that PSCID is required");
         $this->webDriver->get($this->url . "/new_profile/");
 
         $dates = $this->webDriver->findElements(WebDriverBy::cssSelector(".input-date"));
-        $dates[0]->sendKeys("2015-01-01");
-        $dates[1]->sendKeys("2015-02-01");
-        $this->webDriver->findElement(WebDriverBy::Xpath("//*[@id='footer']/div[1]"))
-                 ->click();
+        $days = $this->webDriver->findElements(WebDriverBy::cssSelector(".ui-datepicker-calendar td[data-handler='selectDay']"));
```

Comment:
@kongtiaowang What about this test? It seems to have the same code I added instead of the "executeScript" thing